src/pages/Dashboard/components/AccountList/AccountList.tsx

```diff
@@ -25,19 +25,36 @@ export const AccountList = () => {
   const accountRows = useMemo(() => {
     return (
       <>
-        {Object.values(balances)
-          .sort((a, b) => (bnOrZero(a?.balance).gt(bnOrZero(b?.balance)) ? -1 : 1))
-          .map(account => (
-            <AccountRow
-              key={account.contract ?? account.chain}
-              balance={account.balance ?? '0'}
-              chain={account.chain}
-              tokenId={account.contract}
-            />
-          ))}
+        {Object.keys(balances)
+          .sort((a, b) =>
+            bnOrZero(balances[a]?.balance).gt(bnOrZero(balances[b]?.balance)) ? -1 : 1
+          )
+          .filter(key => bnOrZero(balances[key].balance).gt(0))
```

Comment:
not sure we should filter out zero balances, as users might be interested in tokens they previously held and the associated tx history?

Reply:
This logic already existed. we just moved it up. I'm okay removing it as I agree that a user might be interested in their old assets